### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by newhoggy**
By updating `unagi-chan` dependency, this fixes `aarch64` builds out of the box.

### CODE REVIEW

**Comment by newhoggy** [comment]
```
src/Cardano/Api/Orphans.hs:33:46: error:
    Module ‘Control.Iterate.SetAlgebra’ does not export ‘BiMap(..)’
   |
33 | import           Control.Iterate.SetAlgebra (BiMap (..), Bimap)
   |                                              ^^^^^^^^^^

src/Cardano/Api/Orphans.hs:33:58: error:
    Module ‘Control.Iterate.SetAlgebra’ does not export ‘Bimap’
   |
33 | import           Control.Iterate.SetAlgebra (BiMap (..), Bimap)
   |                                                          ^^^^^

<no location info>: error: [-Wunused-packages, -Werror=unused-packages]
    The following packages were specified via -package or -package-id flags,
    but were not needed for compilation:
      - yaml-0.11.5.0
      - transformers-except-0.1.2
      - parsec-3.1.14.0
      - formatting-6.3.7
      - filepath-1.4.2.1
      - directory-1.3.6.0
      - bech32-1.1.1
      - array-0.5.4.0
      - aeson-pretty-0.8.8
cabal: Failed to build cardano-api-1.31.0 (which is required by
test:tx-generator-test from tx-generator-1.32, exe:tx-generator from
tx-generator-1.32 and others).
```

**Comment by newhoggy** [comment]
```
src/Cardano/Api/LedgerEvent.hs:137:36: error:
    • Couldn't match type ‘Data.Set.Internal.Set
                             (Cardano.Ledger.Shelley.Rewards.Reward (Crypto ledgerera))’
                     with ‘Cardano.Ledger.Coin.Coin’
      Expected type: Map
                       (Cardano.Ledger.Credential.Credential
                          'Cardano.Ledger.Keys.Staking (Crypto ledgerera))
                       (Data.Set.Internal.Set
                          (Cardano.Ledger.Shelley.Rewards.Reward (Crypto ledgerera)))
        Actual type: Map
                       (Cardano.Ledger.Credential.StakeCredential StandardCrypto)
                       Cardano.Ledger.Coin.Coin
    • In the pattern: convertSumRewardsMap -> m
      In the pattern: RewardEvent e (convertSumRewardsMap -> m)
      In the pattern:
        NewEpochEvent (RewardEvent e (convertSumRewardsMap -> m))
    |
137 |     (NewEpochEvent (RewardEvent e (convertSumRewardsMap -> m)))
    |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: cardano-api/src/Cardano/Api/Fees.hs [hunk 1]
```diff
@@ -444,7 +447,7 @@ instance Error TransactionValidityIntervalError where
 
         | otherwise
         = 0 -- This should be impossible.
-
+  displayError (TransactionValidityBasicFailure (Alonzo.UnknownTxIns _)) = "Unknown txins present" -- TODO More information error message
```
Comment (newhoggy) [inline cardano-api/src/Cardano/Api/Fees.hs:450]: How can I better display this error?

Comment (nc6) [reply cardano-api/src/Cardano/Api/Fees.hs:450]: @JaredCorduan ?

Comment (nc6) [reply cardano-api/src/Cardano/Api/Fees.hs:450]: See that this was already updated (but reverted), so you could reuse things there: https://github.com/input-output-hk/cardano-node/pull/3410/files#diff-12e0bb9f3db22ac71c93ed3932f54c91a6f6bf7243afaa29ccd7ecb3657e6069L429

File: cardano-api/src/Cardano/Api/LedgerEvent.hs [hunk 2]
```diff
@@ -153,23 +155,33 @@ pattern LEMirTransfer rp tp rtt ttr <-
...
+                    (convertSumRewardsMapSimple -> tp)
                     (fromShelleyDeltaLovelace -> rtt)
                     (fromShelleyDeltaLovelace -> ttr)
                   )
               )
           )
       )
 
-convertSumRewardsMap ::
+convertRewardEventMapNew ::
```
Comment (nc6) [inline cardano-api/src/Cardano/Api/LedgerEvent.hs:167]: Can we avoid these names? `New` and `Simple` are completely useless and tell me nothing about the difference between these

**Review by nc6 — CHANGES_REQUESTED** [review]
Made a few comments.

**Comment by deepfire** [comment]
Superceded by https://github.com/input-output-hk/cardano-node/pull/3388

**Pull request closed without merging**